cmake/ruy_cc_binary.cmake

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019-2021 Google LLC
```

Comment:
FYI having ending ranges on the years is optional (and IMO let's not make license headers any more complex than they already have to be :stuck_out_tongue:)

Reply:
done